docs/examples/request_data/test_request_data_6.py

```diff
@@ -12,7 +15,14 @@ async def handle_file_upload(
 ) -> str:
     content = await data.read()
     filename = data.filename
-    return f"{filename}, {content.decode()}"
```

Comment:
I changed all tests that used decode like this on a file content as most of the time it will blow up into a 500